recipes/counterfactual_data_augmentation.ipynb

```diff
@@ -27,20 +28,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Authentication\n",
     "import os\n",
     "\n",
-    "# !pip install kili # uncomment if you don't have kili installed already\n",
+    "!pip install --upgrade kili # uncomment if you don't have kili installed already\n",
```

Comment:
```suggestion
    "!pip install --upgrade kili \n",
```

Reply:
Made the changes. Please have a look and resolve if poss.